roles/idm/tasks/prep.yml

```diff
@@ -0,0 +1,37 @@
+---
+
+- name: 'Ensure everything is up-to-date' 
```

Comment:
This may require a reboot of the system. I have a role coming soon that will handle the reboot/wait-for ... so maybe we just leverage that across all playbooks that wants to ensure "eveything" is up-to-date - i.e.: not a responsibility of this role ...